go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/antithesishq/antithesis-sdk-go v0.3.8
 	github.com/ava-labs/avalanchego v1.13.5
 	github.com/ava-labs/firewood-go-ethhash/ffi v0.0.12
-	github.com/ava-labs/libevm v1.13.14-0.3.0.rc.7
+	github.com/ava-labs/libevm v1.13.14-0.3.0.release
```

Comment:
Just confirming this is the correct `libevm` version to bump to, since the `coreth` PR went to `v1.13.15-0.20250904180142-72e9ad796212`?

Reply:
Yeah when the coreth PR was made, we didn't have a proper libevm version. I thought I'd use the non commit version for this PR.